pkg/component/etcd/etcd/crd_test.go

```diff
@@ -17,16 +19,18 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client/fake"
 
 	"github.com/gardener/gardener/pkg/client/kubernetes"
-	"github.com/gardener/gardener/pkg/component"
 	. "github.com/gardener/gardener/pkg/component/etcd/etcd"
+	gardenerutils "github.com/gardener/gardener/pkg/utils/gardener"
 	. "github.com/gardener/gardener/pkg/utils/test/matchers"
 )
 
 var _ = Describe("CRD", func() {
 	var (
 		c           client.Client
 		ctx         = context.TODO()
-		crdDeployer component.Deployer
+		crdDeployer CRDAccess
+		err         error
```

Comment:
Having an err global var in anti-pattern. Can we remove instead of local err vars?

```suggestion
```